e2e/cypress/integration/system_console/site_configuration/announcement_banner_spec.js

```diff
@@ -0,0 +1,107 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+// ***************************************************************
+// - [#] indicates a test step (e.g. # Go to a page)
+// - [*] indicates an assertion (e.g. * Check the title)
+// - Use element ID when selecting an element. Create one if none.
+// ***************************************************************
+
+import {hexToRgbArray, rgbArrayToString} from '../../../utils';
+
+describe('Announcement Banner', () => {
+    it('MM-T1128 Announcement Banner - Dismissible banner shows long text truncated', () => {
+        const bannerText =
+            "Here's an announcement! It has a link: http://example.com. It's a really long announcement, because we have a lot to say. Be sure to read it all, click the link, then dismiss the banner, and then you can go on to the next test, which will have a shorter announcement. Thank you for reading and have a nice day!";
+        const bannerBgColor = '#4378da';
+        const bannerBgColorRGBArray = hexToRgbArray(bannerBgColor);
+        const bannerTextColor = '#ffffff';
+        const bannerTextColorRGBArray = hexToRgbArray(bannerTextColor);
+
+        // # Login as System Admin
+        cy.apiAdminLogin();
+
+        // # Go to announcement banner page of system console
+        cy.visit('/admin_console/site_config/announcement_banner');
+
+        // # Enable banner if not already enabled
+        cy.findByTestId('AnnouncementSettings.EnableBanner').within(() => {
+            cy.findByText('true').should('be.visible').click({force: true});
+        });
+
+        // # Enter the long banner text
+        cy.findByTestId('AnnouncementSettings.BannerText').within(() => {
+            cy.get('input').should('be.visible').clear().type(bannerText);
```

Comment:
when i wrote the invoke type for entering the text, the input text was vanishing as soon as input lost focus, 
so i had to resort to type methods

Reply:
figured it out, had to add type at the end